include/LightGBM/meta.h

```diff
@@ -17,10 +17,10 @@ namespace LightGBM {
 /*! \brief Type of data size, it is better to use signed type*/
 typedef int32_t data_size_t;
 
-// Enable following marco to use double for score_t
+// Enable following macro to use double for score_t
 // #define SCORE_T_USE_DOUBLE
 
-// Enable following marco to use double for label_t
+// Enable following Macao to use double for label_t
```

Comment:
🤣 

Is it a spoiler with your flight destination?

Reply:
haha no! Just me clearly mistyping my selection in `aspell`. Thanks for catching 😆